service-python/pdfgenerator/src/lib/templates/hypertension/rating_schedule.html

```diff
@@ -1,4 +1,4 @@
-<h3>Hypertension Rating Schedule</h3>
+<h2>Hypertension Rating Schedule</h2>
```

Comment:
This change does not seem to have an effect on the text size in the PDF. I tried putting in the <h1> or <h6> and it looks the same.

Reply:
Hmm not sure what the issue might be. I reran it after making the same updates and got the changes
![image](https://user-images.githubusercontent.com/13013464/189709496-5bea30d7-dc3c-4d37-930c-a931a1b4c12d.png)
![image](https://user-images.githubusercontent.com/13013464/189709605-d0027e46-9639-4ca9-9d8f-ab23e090167c.png)
![image](https://user-images.githubusercontent.com/13013464/189709660-d3aa7e35-5f2b-4ee5-89b7-b0095a95eba0.png)

Reply:
Interesting, I tried rebuilding a couple times but I will try again

Reply:
![Screen Shot 2022-09-12 at 1 46 54 PM](https://user-images.githubusercontent.com/54538359/189721758-68a9750d-b0f3-47af-af02-994a0273b443.png)
For me it still looks like this, not sure why